src/components/toast-notifications/toast-notifications.html

```diff
@@ -0,0 +1,19 @@
+<div class="toast-notifications-list-pf">
+  <div ng-repeat="(alertID, alert) in (alerts | filterCollection : filter) track by (alertID + (alert.message || alert.details))" ng-if="!alert.hidden"
+       ng-mouseenter="setHover(alert, true)" ng-mouseleave="setHover(alert, false)">
+    <div class="toast-pf alert {{alert.type | alertStatus}}" ng-class="{'alert-dismissable': !hideCloseButton}">
+      <button ng-if="!hideCloseButton" type="button" class="close" ng-click="close(alert)" aria-hidden="true">
```

Comment:
instead of making the entire button aria-hidden, the aria-hidden should be on the icon below, and an sr-only span should have "Close" text, so that a screen reader user could still dismiss this toast notification